asset/script/main.js

```diff
@@ -0,0 +1,19 @@
+const mobileMenu = document.querySelector('.mobile-menu');
+const hamburgar = document.getElementById('hamburger-icon');
+const navClose = document.querySelectorAll('.mobile-nav-link');
+
+function toggleMobileMenu() {
+  mobileMenu.classList.toggle('open');
+  hamburgar.classList.toggle('change');
+}
+
+function closeNav() {
+  hamburgar.classList.remove('change');
+  mobileMenu.classList.remove('open');
+}
+
+hamburgar.addEventListener('click', toggleMobileMenu);
+
+navClose.forEach((elem) => {
+  elem.addEventListener('click', closeNav);
+});
```

Comment:
While you did a good job Implementing this function unfortunately your hambuger button isnt visible. Please make sure the hamburger is visible when user switches to the mobile view.

Look at your design:
![Screenshot from 2022-05-10 15-37-31](https://user-images.githubusercontent.com/79506105/167620404-766c2038-055f-4052-b871-0880a8e06bb1.png)

Your app should look like this: 
![Screenshot from 2022-05-10 15-47-05](https://user-images.githubusercontent.com/79506105/167621275-20047664-72bd-4593-bc6f-eb94892551d7.png)

And then after the hamburger is clicked it should display this:
![Screenshot from 2022-05-10 15-47-19](https://user-images.githubusercontent.com/79506105/167621430-d0b16fe9-2595-409a-9ff6-6dc5f654a9cc.png)